src/main/resources/admin/tools/content-type-manager/components/input/fields/component-input-fields-config.html

```diff
@@ -86,38 +105,41 @@
         json() {
           return this.$store.getters.jsonPath(this.path);
         },
+        elements() {
```

Comment:
Just an idea. Feel free to ignore. 
Would it be faster to go over all elements once and then set the element into the respective group? 
Instead of filtering it once for each group?

Reply:
Hmm I think it can't be like that, because:
- They are dynamic, therefore if it's not set as data or computed prop. of a component, it'll not be trigger reactivity;
- Elements can be added and removed (add input / remove input for example), so it'd be necessary to run that function to set elements of each group multiple times;